membership-attribute-service/app/controllers/PaymentUpdateController.scala

```diff
@@ -169,10 +171,16 @@ class PaymentUpdateController(
             lastName = billToContact.lastName,
             countryCode = "GB",
           )
+          paymentGatewayToUse =
+            if (paymentGatewayOwner.equals("tortoise-media")) {
```

Comment:
what type is `paymentGatewayOwner`?  I would expect it to be an `Option[String]` in which case we would want to do 
```suggestion
            if (paymentGatewayOwner.contains("tortoise-media")) {
```
I'm not sure if `.equals` is the same as `==` otherwise, but it might only check the type at run time rather than compile time.

Reply:
@johnduffell Using some AI:

> These two conditions are not the same in Scala.
> 
> `paymentGatewayOwner.equals("tortoise-media")`
> This checks for exact string equality
> Returns true only if paymentGatewayOwner is precisely "tortoise-media"
> 
> 
> `paymentGatewayOwner.contains("tortoise-media")`
> This checks if the string contains "tortoise-media" as a substring
> Returns true if paymentGatewayOwner is "tortoise-media" but also if it's something like "the-tortoise-media", "tortoise-media-company", etc.
> 
> The contains method is more permissive since it will match partial occurrences, while equals requires an exact match.

Reply:
Should we trust AI on this one? Because I'm afraid of using other values in the future that may include "tortoise-media". Also, should we create some sort of enum here?